ToolsTest/Володимир/xUnit.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using Tools;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+using Xunit;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+using nAssert = NUnit.Framework.Assert;
+using xAssert = Xunit.Assert;
+using Theory = Xunit.TheoryAttribute;
+using Tools.Models;
+
+namespace ToolsTest.Володимир
+{
+    public class xUnit
+    {
+        [Fact]
+        public void TryReplaceText()
```

Comment:
This test is failed. Input row should contain the "old" text for passing the test.